ui/src/app/pages/access/access-renewal.spec.tsx

```diff
@@ -27,7 +27,7 @@ describe('Access Renewal Page', () => {
     const {profile} = profileStore.get();
 
     const newProfile = fp.set('accessModules',
-        {modules: expirableAccessModules.map(m => {return {moduleName: m, expirationEpochMillis: expiredTime}})},
+        {modules: Array.from(accessRenewalTitles.keys()).map(m => {return {moduleName: m, expirationEpochMillis: expiredTime}})},
```

Comment:
FYI: I just checked: Map guarantees keys to be returned in insertion order, so this should be determinstic.

Reply:
oh good, thanks